docs/crio.8.md

```diff
@@ -120,6 +120,8 @@ crio [GLOBAL OPTIONS] config [OPTIONS]
 
 **--cpu-profile**: Set the CPU profile file path
 
+**--allow-strace**: Allow strace for containers when the strace annotation/label "io.kubernetes.cri-o.Strace=true" is set
```

Comment:
Except for the last few of these, the options in this man page seem to be ordered alphabetically.  There are a number of them, so I think we may want to continue with that alphabetization to keep them easy to find.